cfy_manager/components/rabbitmq/config/erl_inetrc

```diff
@@ -0,0 +1,3 @@
+%% Tells DNS client on RabbitMQ nodes and CLI tools to resolve hostnames to IPv6 addresses.
+%% The trailing dot is not optional.
+{inet6,true}.
```

Comment:
Should we use the ipv6_enabled flag here as well? (should we even use it below?)

Reply:
OK, ignore this, I see you actually do the check in code before deploying this file.

Reply:
No, not necessarily.  This file is deployed only in case `ipv6_enabled` is enabled.  https://github.com/cloudify-cosmo/cloudify-manager-install/blob/4f4d921cac645b606e6470da3e559fb13b7223de/cfy_manager/components/rabbitmq/rabbitmq.py#L98-L100